src/utils/script_args.py

```diff
@@ -18,6 +18,7 @@ class ScriptArgs:
     dune: DuneFetcher
     post_tx: bool
     dry_run: bool
+    consolidate_transfers: bool
```

Comment:
Nit: Not a big fan of having the args in a separate file. In my opinion using something like [click](https://click.palletsprojects.com/en/8.1.x/) would be something I favor more. Mostly due to the fact that I don't have to check for which script these args are used for and I can see all the inputs in `transfer_file.py`.

Reply:
I could move this data model into the transfer file (since its only used here), but I don't believe this is the place to be refactoring the location of things that are already established. If you would like to see this implemented differently, it might be better to create an issue...

Reply:
I agree; this isn't the place to implement it. I just saw it and wanted to bring it up.

I think we should continue this discussion on a separate issue.